src/constants.ts

```diff
@@ -1,3 +1,11 @@
+import { Currency } from "src/domain";
+
+export const STORAGE_VERSION_KEY = "version";
+export const STORAGE_VERSION = 1;
+
+export const PREFERRED_CURRENCY_KEY = "currency";
+export const PREFERRED_CURRENCY = Currency.USD;
```

Comment:
Maybe `PREFERRED_CURRENCY_DEFAULT`, `DEFAULT_PREFERRED_CURRENCY` or something similar? Just to emphasize that it's the default value.

Reply:
I have consciously omitted the prefix `DEFAULT` since it's implicit in the context of the constants. I mean, do we really need this emphasis?